app/src/main/java/edu/akelael/comics/model/ComicListModel.java

```diff
@@ -0,0 +1,48 @@
+package edu.akelael.comics.model;
+
+import java.util.HashMap;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+public class ComicListModel implements IComicListModel {
+    Server server;
+    Retrofit retrofit;
+    HashMap<String, String> mMap;
+
+    @Override
+    public void initializeApiConnection(){
+        Retrofit.Builder retrofitBuilder;
+        retrofitBuilder = new Retrofit.Builder();
+        retrofitBuilder.baseUrl(server.baseUrl);
+        retrofitBuilder.addConverterFactory(GsonConverterFactory.create());
+        retrofit = retrofitBuilder.build();
+        mMap = new HashMap<>();
+        mMap.put("ts", "ts");
+        mMap.put("apikey", "apikey");
+        mMap.put("hash", "hash");
```

Comment:
I see 2 different things in this method (though both of them make sense to be here IMO). First, there is the creation of the "retrofit" object (or server object, if the suggestion I made before is applied). Then, there is the initialization of the hash map with the request content. What I believe it would improve readability in this case is to move these two things to two separate methods. That way, the initializeApiConnection could be read something like:

```java
public void initializeApiConnection() {
    intializeServer(); // not the best name IMO
    initializeRequestContent();
}
```